.travis/script.sh

```diff
@@ -8,48 +8,16 @@ die() {
 set -x
 
 [ $TRAVIS_OS_NAME != linux ] || {
-    export PATH=/tmp/arc_gnu_2017.09_prebuilt_elf32_le_linux_install/bin:$PATH || die
+    export PATH=/tmp/arc_gnu_${GNU_VER}_prebuilt_elf32_le_linux_install/bin:$PATH || die
     git checkout -- . || die
     bash apply_embARC_patch.sh || die
     cd .travis || die
 
-    [ $TOOLCHAIN != sphinx -o $BOARD != none -o $BD_VER != none -o $CUR_CORE != none ] || {
+    [ $TOOLCHAIN != sphinx -o $BOARD != none -o $OSP_ROOT != none -o $GNU_VER != none ] || {
         bash deploy_doc.sh || die
     }
+    [ $TOOLCHAIN == sphinx -o $BOARD != emsk -o $OSP_ROOT == none -o $GNU_VER == none ] || {
```

Comment:
[ $TOOLCHAIN == sphinx -o $BOARD != emsk -o $OSP_ROOT == none -o $GNU_VER == none ] 
Please not limited to just EMSK board, check this part